frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableDetail/Indices/Indices.tsx

```diff
@@ -1,40 +1,29 @@
 import type { Indices as IndicesType } from '@liam-hq/db-structure'
-import { Hash } from '@liam-hq/ui'
-import clsx from 'clsx'
+import { FileText } from '@liam-hq/ui'
 import type { FC } from 'react'
 import { CollapsibleHeader } from '../CollapsibleHeader'
-import styles from './Indices.module.css'
+import { IndexesItem } from './IndexesItem'
 
 type Props = {
   indices: IndicesType
 }
 
 export const Indices: FC<Props> = ({ indices }) => {
-  // NOTE: 17px is the height of one item in the list
-  // 24px is the padding of the list
-  // 1px is the border of the list
-  const contentMaxHeight = Object.keys(indices).length * 17 + 24 + 1
+  const contentMaxHeight = Object.keys(indices).length * 400
```

Comment:
I want to check if there is any problem after deploying it in the preview environment.